Susi/Controllers/ResetPasswordController/ResetPasswordVCMethods.swift

```diff
@@ -17,7 +17,7 @@ extension ResetPasswordViewController {
 
     func setupView() {
         navigationItem.leftViews = [backButton]
-        navigationItem.title = ControllerConstants.resetPassword
+        title = ControllerConstants.resetPassword
```

Comment:
Does this require adding `.localized()` as well? Please verify other places as well